xml/operations-configuring-identity-identity_limitations.xml

```diff
@@ -322,8 +322,17 @@ for production use until a future OpenStack release.</li> -->
    </listitem>
    <listitem>
     <para>
-     Only WebSSO federation using the SAML method is supported in
-     &product; . OpenID-based federation is not currently supported.
+     The &productname; WebSSO function with OpenID method was tested with
+     Google OAuth 2.0 APIs, which conforms to the OpenID Connect specification.
+     The interaction between Keystone and external Identity Provider (IDP) is
```

Comment:
It reads better if we say "and the external Identity Provider (IdP)".   (Add "the" and use a lowercase letter d in "IdP" like we do below to be consistent). I see both IdP and IDP but it seems IdP is more common and in my opinion more correct because it's not spelled "IDentity". I don't care that much which we use as long as we're consistent.

Reply:
good catch!